sig-release/README.md

```diff
@@ -0,0 +1,122 @@
+# SIG Release
+
+### Table of Contents
+- [Purpose](#purpose)
+- [Join us!](#join-us)
+- [Broad responsibilities](#broad-responsibilities)
+- [Specific responsibilities](#specific-responsibilities)
+- [Relation to the Release Manager and Team](#relation-to-the-release-manager-and-team)
+- [Ongoing responsibilities of the Release Team](#ongoing-responsibilities-of-the-release-team)
+  - ["Minor" Releases](#minor-releases)
+  - [“Patch” releases](#patch-releases)
+  - ["Security" releases](#security-releases)
+- [References](#references)
+
+## Join us!
+- [Google group](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+- [Slack channel](https://kubernetes.slack.com/messages/C56HWQ0TH/)
```

Comment:
There is also #k8s-release as used by the release team, should that be decommissioned?